### DISCUSSION TITLE
Contributing Guidelines for newcomers

### DISCUSSION THREAD

**Discussion by zaleslaw** [Kotlin/kotlindl#134] [General]
We released the [Contributing Guidelines](https://github.com/JetBrains/KotlinDL/blob/master/CONTRIBUTING.md) for newcomers.

**Comment by zaleslaw** [comment]
Please, @avan1235 @mkaze @therealansh @devcrocod @d-lowl @dosier @kokorins spend some minutes and read.
There is a common practice for kotlinx projects and based on [kotlinx.serialization](https://github.com/Kotlin/kotlinx.serialization) guidelines with the addition of our informal practice.

Hope it will be helpful for you and new contributors.

**Reply by d-lowl** [reply]
Looks pretty good. Potential suggestion. Since it says (or used to say?) on a few issues for layer implementation "if possible, create a small trainable network with it (in your own GitHub) and attach a link here in the comments", it might be good to have a boilerplate repo, so devs can fork and use it for those examples.

**Reply by zaleslaw** [reply]
Great idea to have a skeleton/template repo, thanks for the addition
